File: Benchmarks/enterprise/python/flask/testcode/benchmark_test_71172.py
# SPDX-License-Identifier: Apache-2.0
from flask import request, jsonify
import os


def BenchmarkTest71172():
    json_value = (request.get_json(silent=True) or {}).get('payload', '')
    data = f'{json_value:.200s}'
    os.environ['APP_USER_PREFERENCE'] = str(data)
    return jsonify({'config_set': 'APP_USER_PREFERENCE'}), 200

File: Benchmarks/enterprise/java/spring/testcode/BenchmarkTest71172.java
// SPDX-License-Identifier: Apache-2.0
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
import java.net.*;
import org.springframework.web.bind.annotation.*;

@RestController
public class BenchmarkTest71172 {

    @GetMapping("/BenchmarkTest71172")
    public void BenchmarkTest71172(@CookieValue("session_token") String sessionToken, HttpServletRequest request, HttpServletResponse response) throws Exception {
        String cookieValue = sessionToken != null ? sessionToken : "";
        String data = String.format("payload=%s", cookieValue);
        new Socket(data, 80).close();
        response.setContentType("application/json");
        response.getWriter().print("{\"id\":0}");
    }
}
